### PR TITLE
Adding script to setup GPU usage

### PR DESCRIPTION
**Pull request by lememta**

### CODE REVIEW

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -17,10 +17,12 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: 3.6
-    - name: Install
-      run:
-        pip install -e '.[tf]'
-        pip install -r test-requirements.txt
+    - name: Install TF 
+      run:  pip install tensorflow==2.4.0
```
Comment (johntmyers) [inline .github/workflows/integration-tests.yml:21]: Change to `2.4` to be consistent with the shell script command

File: .github/workflows/unit-tests.yml [hunk 2]
```diff
@@ -11,10 +11,12 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: 3.6
-    - name: Install
-      run:
-        pip install -e '.[tf]'
-        pip install -r test-requirements.txt
+    - name: Install TF 
+      run:  pip install tensorflow==2.4.0
```
Comment (johntmyers) [inline .github/workflows/unit-tests.yml:15]: Change to 2.4. to be consistent with the shell script command

File: README.md [hunk 3]
```diff
@@ -54,6 +51,24 @@ $ jupyter notebook
 
 When the UI launches in your browser, navigate to `examples/synthetic_records.ipynb` and get generating!
 
+If you want to install `gretel-synthetics` locally and use GPU:
```
Comment (johntmyers) [inline README.md:54]: "use a GPU (recommended)."

File: README.md [hunk 4]
```diff
@@ -54,6 +51,24 @@ $ jupyter notebook
...
+
+2. Activate the virtual environment
+
+```
+$ conda activate tf
+```
+
+3. Run the setup script `./setup-utils/setup-gretel-synthetics-tensorflow24-with-gpu.sh`
+
+The last step will install all the necessary software packages for GPU, `tensorflow=2.4` and `gretel-synthetics`.
```
Comment (johntmyers) [inline README.md:70]: "for GPU usage"

also `tensorflow==2.4`

File: setup-utils/setup-gretel-synthetics-tensorflow24-with-gpu.sh [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+sudo apt-get update
+
+wget http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64/nvidia-machine-learning-repo-ubuntu1804_1.0.0-1_amd64.deb
+
+sudo apt install ./nvidia-machine-learning-repo-ubuntu1804_1.0.0-1_amd64.deb
+sudo apt-get update
+
+# Install NVIDIA driver
+sudo apt-get install --no-install-recommends nvidia-driver-450
+# Reboot. Check that GPUs are visible using the command: nvidia-smi
```
Comment (johntmyers) [inline setup-utils/setup-gretel-synthetics-tensorflow24-with-gpu.sh:19]: Does a reboot here really need to happen? Does this work w/o rebooting mid-way through?

**Review by johntmyers — CHANGES_REQUESTED** [review]

File: setup-utils/setup-gretel-synthetics-tensorflow24-with-gpu.sh (hunk 5 above)
Comment (lememta) [reply setup-utils/setup-gretel-synthetics-tensorflow24-with-gpu.sh]: confirmed that you don't need to reboot

**Review by johntmyers — APPROVED** [review]

**Pull request merged**